docs/document-parts.md

```diff
@@ -75,70 +152,63 @@ parts:
 
 The advantage of this method is that the content is not rendered in your document.
 
-(parts:implicit)=
-
-## Implicit Parts using a Title
-
-If you are rendering your project in other places, it can be helpful to leave these sections directly in the document.
-Complete this using a header as usual:
-
-```
-# Abstract
-
-This is my abstract!
-```
-
-Note that frontmatter parts and explicitly tagged cells/blocks will take precedence over this method. Themes may choose to only pick up a subset of implicit parts, for example, only an `Abstract` and not `Summary` as summary section can be used in other contexts.
-
-(parts:blocks)=
-
-## In a Jupyter Notebook cells and blocks
+(parts:site)=
 
-When using a Jupyter Notebook, you can add a `tag` to the cell with the part name, if multiple cells share that tag, they will be extracted and merged. This can also be represented in a [block](./blocks.md):
+## Add parts to your website
 
-```markdown
-+++ { "part": "abstract" }
+[Website themes](./website-templates.md) have their own configuration for parts because they often have extra user interface elements that are not part of a standard "document" structure. These are theme-dependent. For example, the [default myst themes](#default-web-themes) support [a website footer](#navigation:footer).
```

Comment:
```suggestion
[Website themes](./website-templates.md) have additional parts because they render user interface elements that are not part of a standard "document" structure. These are theme-dependent: for example, the [default myst themes](#default-web-themes) support a [`footer` part](#navigation:footer).
```